react-graph/src/Plot.tsx

```diff
@@ -537,7 +560,7 @@ const Plot: React.FunctionComponent<IProps> = (props) => {
                        <InteractiveButtons showPan={(props.pan === undefined || props.pan)}
                         showZoom={props.zoom === undefined || props.zoom}
                         showReset={!(props.pan !== undefined && props.zoom !== undefined && !props.zoom && !props.pan)}
-                        showSelect={props.onSelect !== undefined || handlers.current.size > 0}
+                        showSelect={props.onSelect !== undefined || props.onSelectConsumable !== undefined || handlers.current.size > 0}
```

Comment:
I still don't think we need a `onSelectConsumable` in the `IProps` nothing outside `react-graph` should have to deal with this